### ISSUE TITLE
adding support for bonding in iOS

### ISSUE THREAD

**Issue by Antorai78** [randdusing/cordova-plugin-bluetoothle#724]
Is it possible to pair a bluetooth device using the bonding procedure on iOS as it happens on Android by implementing the BluetoothDevice.createBond method?

**Comment by JesperBalslev** [comment]
No, afaik you need to initialize the bonding from your firmware on iOS.
